Lib/_pyio.py

```diff
@@ -1794,6 +1794,20 @@ def isatty(self):
         self._checkClosed()
         return os.isatty(self._fd)
 
+    def _isatty_openonly(self):
```

Comment:
I suggest to rename this method to:

```suggestion
    def _isatty_open_only(self):
```

since "open" and "only" are two words. Same remark for the C code.